src/server/handlers/github.js

```diff
@@ -0,0 +1,96 @@
+import { conf } from '../helpers/config';
+import request from 'request';
+
+const GITHUB_USERNAME = conf.get('GITHUB_USERNAME');
+const GITHUB_PASSWORD = conf.get('GITHUB_PASSWORD');
+const GITHUB_API_ENDPOINT = conf.get('GITHUB_API_ENDPOINT')
+const HTTP_PROXY = conf.get('HTTP_PROXY');
+
+const RESPONSE_NOT_FOUND = {
+  status: "error",
+  payload: {
+    message: "github-repository-not-found"
+  }
+};
+
+const RESPONSE_AUTHENTICATION_FAILED = {
+  status: "error",
+  payload: {
+    message: "github-authentication-failed"
+  }
+};
+
+const RESPONSE_OTHER = {
+  status: "error",
+  payload: {
+    message: "github-error-other"
+  }
+}
+
+const RESPONSE_ALREADY_CREATED = {
+  status: "error",
+  payload: {
+    message: "github-already-created"
+  }
+}
+
+const RESPONSE_CREATED = {
+  status: "success",
+  payload: {
+    message: "github-webhook-created"
+  }
+}
```

Comment:
I'm interested in your comments regarding JSON responses. It would be nice if we could standardise responses for all our Express endpoints.

The initial standard I've created here is loosely based on what we did at The Last Place: meta data about the request at the top level (status etc) and the payload of the response (message etc). I'm open to suggestions and alternatives.